specification.markdown

```diff
@@ -97,242 +170,298 @@ followed by the Data Size, and then the non-interpreted Binary itself:
+
+Element Name:   EBMLMaxIDLength
+
+    Level:          1
+    Class-ID:       [42][F2]
+    Mandatory:      Yes
+    Multiple:       No
+    Range:          -
+    Default:        4
+    Element Type:   Unsigned Integer
+    Description:    The maximum length of the Element IDs you'll find in
+                    this file.
+
+Element Name:   EBMLMaxSizeLength
+
+    Level:          1
+    Class-ID:       [42][F3]
+    Mandatory:      Yes
+    Multiple:       No
+    Range:          -
+    Default:        8
+    Element Type:   Unsigned Integer
+    Description:    The maximum length of the sizes you'll find in this 
+                    file. This does not override the element size
+                    indicated at the beginning of an element. Elements
+                    that have an indicated size which is larger than
+                    what is allowed by EBMLMaxSizeLength SHALL be
+                    considered invalid.
+
+Element Name:   DocType
+
+    Level:          1
+    Class-ID:       [42][82]
+    Mandatory:      Yes
+    Multiple:       No
+    Range:          -
+    Default:        -
+    Element Type:   String
+    Description:    A string that describes the type of document that
+                    follows this EBML header.
+
+Element Name:   DocTypeVersion
+
+    Level:          1
+    Class-ID:       [42][87]
+    Mandatory:      Yes
+    Multiple:       No
+    Range:          -
+    Default:        1
+    Element Type:   Unsigned Integer
+    Description:    The version of DocType interpreter used to create
+                    the file.
+
+Element Name:   DocTypeReadVersion
+
+    Level:          1
+    Class-ID:       [42][85]
+    Mandatory:      Yes
+    Multiple:       No
+    Range:          -
+    Default:        1
+    Element Type:   Unsigned Integer
+    Description:    The minimum DocType version an interpreter has to
+                    support to read this file.
+
+### Global elements (used everywhere in the format)
+
+Element Name:   CRC-32
+
+    Level:          1+
+    Class-ID:       [BF]
+    Mandatory:      No
+    Multiple:       No
+    Range:          -
+    Default:        -
+    Element Type:   Binary
+    Description:    The CRC is computed on all the data from the last
+                    CRC element (or start of the upper level element),
+                    up to the CRC element, including other previous CRC
+                    elements. All level 1 elements SHOULD include a
+                    CRC-32.
+
+Element Name:   Void
+
+    Level:          1+
+    Class-ID:       [EC]
+    Mandatory:      No
+    Multiple:       No
+    Range:          -
+    Default:        -
+    Element Type:   Binary
+    Description:    Used to void damaged data, to avoid unexpected
+                    behaviors when using damaged data. The content is
+                    discarded. Also used to reserve space in a
+                    sub-element for later use.
+
+### signature
```

Comment:
In the Element Template, we could add a value called "Element Status", then for each signature value, we can set it to "Deprecated since EBML version 1". Any better idea?